fail2ban/server/filterpoll.py

```diff
@@ -78,6 +79,16 @@ def _delLogPath(self, path):
 		del self.__file404Cnt[path]
 
 	##
+	# Get a modified log path at once
+	#
+	def getModified(self, modlst):
+		for container in self.getLogPath():
+			filename = container.getFileName()
+			if self.isModified(filename):
+				modlst.append(filename)
+		return modlst
```

Comment:
just as planned: `getModified` will be executed in `wait_for`, so expected a return value (to check it in lambda), and it expected `modlst` should be changed hereafter..., see https://github.com/fail2ban/fail2ban/pull/1116/files#diff-7060c52d9c93892537b937c9329e7cf7R107
BTW: `modlst = modlst + [filename]` is much slower as `modlst.append`